### PR TITLE
Update issue templates

### LINKED ISSUE #1199: Update issue templates under .github
### Description

Like other projects such as python-slack-sdk, bolt-python, and java-slack-sdk, we may want to have multiple issue templates per category

Example: https://github.com/slackapi/python-slack-sdk/tree/main/.github/ISSUE_TEMPLATE

### What type of issue is this? (place an `x` in one of the `[ ]`)
- [ ] bug
- [x] enhancement (feature request)
- [ ] question
- [ ] documentation related
- [ ] testing related
- [x] discussion

### Requirements (place an `x` in each of the `[ ]`)
* [x] I've read and understood the [Contributing guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

### PR DESCRIPTION
**Pull request by srajiang**
Fixes #1199. Adds issue templates in the new recommended format.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+(Filling out the following with as much detail as you can provide will help us solve your issue sooner.)
+
+#### Packages:
+
+Select all that apply:
+
+- [ ] `@slack/web-api`
+- [ ] `@slack/events-api`
+- [ ] `@slack/interactive-messages`
```
Comment (seratch) [inline .github/ISSUE_TEMPLATE/bug.md:18]: These are EOL-ed. Can you remove them?

File: .github/ISSUE_TEMPLATE/enhancement---feature-request.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+(Describe your issue and goal here) 
+
+#### Packages:
+
+Select all that apply:
+
+- [ ] `@slack/web-api`
+- [ ] `@slack/events-api`
+- [ ] `@slack/interactive-messages`
```
Comment (seratch) [inline .github/ISSUE_TEMPLATE/enhancement---feature-request.md:18]: same as above

File: .github/ISSUE_TEMPLATE/question.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+(Describe your issue and goal here)
+
+#### Packages:
+
+Select all that apply:
+
+- [ ] `@slack/web-api`
+- [ ] `@slack/events-api`
+- [ ] `@slack/interactive-messages`
```
Comment (seratch) [inline .github/ISSUE_TEMPLATE/question.md:18]: same

**Review by seratch — COMMENTED** [review]
Thanks for working on this! Can you remove events-api and interactive-messages from the package list? They are already EOL-ed.

**Comment by srajiang** [comment]
@seratch Removed the EOL'ed packages!

**Review by seratch — APPROVED** [review]
LGTM!

**Pull request merged**